### ISSUE TITLE
Train on single-channel images

### ISSUE THREAD

**Issue by njuhang** [xinntao/BasicSR#378]
The loss function requires a vgg network, but there is no pre-trained vgg network for single-channel images. How to deal with this problem?
损失函数需要一个vgg网络，但是对于单通道的图像没有预训练的vgg网络，如何处理这个问题呢？

**Comment by jnjaby** [comment]
> The loss function requires a vgg network, but there is no pre-trained vgg network for single-channel images. How to deal with this problem?

Hi, this can be solved by duplicating the gray channel to make a 3-channel image. Now the image has 3 color channels but it still looks grey, and it can be fed into the VGG network.

**Comment by YAN287-hub** [comment]
> > 损失函数需要VGG网络，但目前没有针对单通道图像预训练的VGG网络。如何解决这个问题？
> 
> 您好，这个问题可以通过复制灰度通道来生成三通道图像。现在图像有了三个颜色通道，但看起来仍然是灰色的，可以将其输入到 VGG 网络中。

你好，具体在那个文件中改啊，我怎么没找到呢
